rules/lowercase_description.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const ruleMsg = 'it() description should begin with lowercase';
+
+const isItOrTestorDescribeFunction = node => {
+  return (
+    node.type === 'CallExpression' &&
+    node.callee &&
+    (node.callee.name === 'it' ||
+      node.callee.name === 'test' ||
+      node.callee.name === 'describe')
+  );
+};
+
+const isItDescription = node => {
+  const foo =
+    node.arguments &&
+    node.arguments[0] &&
+    (node.arguments[0].type === 'Literal' ||
+      node.arguments[0].type === 'TemplateLiteral');
+  return foo;
+};
+
+const testDescription = node => {
+  const { type } = node.arguments[0];
```

Comment:
Object destructuring is not supported on Node 4, which is causing the Node 4 failure on Travis.

Related: https://github.com/jest-community/eslint-plugin-jest/pull/64#discussion_r167413883